packages/mobile/src/onboarding/registration/SelectCountry.tsx

```diff
@@ -0,0 +1,86 @@
+import KeyboardSpacer from '@celo/react-components/components/KeyboardSpacer'
+import SearchInput from '@celo/react-components/components/SearchInput'
+import colors from '@celo/react-components/styles/colors.v2'
+import { LocalizedCountry } from '@celo/utils/src/countries'
+import { StackScreenProps } from '@react-navigation/stack'
+import React, { useCallback, useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { FlatList, StyleSheet, View } from 'react-native'
+import { useSafeArea } from 'react-native-safe-area-view'
+import { useDispatch } from 'react-redux'
+import { Namespaces } from 'src/i18n'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import SelectCountryItem from 'src/onboarding/registration/SelectCountryItem'
+
+const keyExtractor = (item: LocalizedCountry) => item.alpha2
+
+type Props = StackScreenProps<StackParamList, Screens.SelectCountry>
+
+export default function SelectCountry({ navigation, route }: Props) {
+  const { t } = useTranslation(Namespaces.accountScreen10)
+  const { countries, selectedCountryCodeAlpha2 } = route.params
+  const dispatch = useDispatch()
+  const [searchText, setSearchText] = useState('')
+
+  const filteredCountries = useMemo(() => countries.getFilteredCountries(searchText), [
+    countries,
+    searchText,
+  ])
+
+  function onChangeText(text: string) {
+    setSearchText(text)
+  }
+
+  const onSelect = useCallback(
+    (country: LocalizedCountry) => {
+      navigation.navigate(Screens.JoinCelo, { selectedCountryCodeAlpha2: country.alpha2 })
+    },
+    [dispatch]
+  )
```

Comment:
I think here we can do:
```suggestion
function onSelect(country: LocalizedCountry) {
   navigation.navigate(Screens.JoinCelo, { selectedCountryCodeAlpha2: country.alpha2 })
 }
```

Reply:
Indeed 👍